sdk/keyvault/azure-security-keyvault-certificates/CHANGELOG.md

```diff
@@ -1,7 +1,7 @@
 # Release History
 
-## 4.0.0-beta.1 (Unreleased)
+## 4.0.0-beta.1 
 
 ### New Features
 
-- Added `CertificateClient` with  `GetCertificate()` and `GetCertificateVersion()` APIs.
+- initial preview
```

Comment:
We should add more text to highlight what APIs and capabilities we are adding as part of this release, so that customers know what's new here and how they can benefit.

Reply:
Something that showcases the header to include (the simplified header), list of commonly used APIs, and link to relevant conceptual docs:
https://docs.microsoft.com/en-us/azure/key-vault/certificates/certificate-scenarios
https://docs.microsoft.com/en-us/rest/api/keyvault/#certificate-operations

See examples like other "first releases":
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/storage/azure-storage-blobs/CHANGELOG.md#new-features-6
https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/keyvault/azure-security-keyvault-keys/CHANGELOG.md#new-features-1

Reply:
@vhvb1989, we probably wanna do something similar/verbose in keyvault secrets, so users have more info of what support was added. Maybe for the next beta we can do that.

https://github.com/Azure/azure-sdk-for-cpp/blob/main/sdk/keyvault/azure-security-keyvault-secrets/CHANGELOG.md#400-beta1-2021-09-08